.buildkite/pipeline.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+steps:
+  - label: "build-gnu-x86-bzImage"
+    commands:
+     - cargo build --release --features bzImage
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "rustvmm/dev:v2"
+          always-pull: true
+
+  - label: "build-musl-x86"
+    commands:
+     - cargo build --release --features bzImage --target x86_64-unknown-linux-musl
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "rustvmm/dev:v2"
+          always-pull: true
+
+  - label: "unittests-gnu-x86-bzImage"
```

Comment:
This one is not needed because we are unit testing all the features with the default pipeline from rust-vmm-ci. Same for musl unittests.